components/core/LanguageSwitcher.vue

```diff
@@ -19,14 +19,16 @@ export default {
     ModalSwitcher
   },
   data() {
-    const storeView = currentStoreView();
     return {
-      country: storeView.i18n.defaultCountry,
-      lang: storeView.i18n.defaultLanguage,
-      currency: storeView.i18n.currencyCode,
       loadLanguagesModal: false
     };
   },
+  computed: {
+    linkName() {
+      const { i18n = {} } = currentStoreView();
```

Comment:
Moim zdaniem wywołanie funkcji `currentStoreView` najlepiej przenieść poza export i tam przypisać aktualny storeView do jakiejś stałej. W tym przypadku, jak to jest w `computed`, to ta funkcja wywoływana jest mnóstwo razy (za każdym razem, jak computed jest przeliczane).

Reply:
done :)